tests/Auth0.ManagementApi.IntegrationTests/LogStreamsTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Auth0.Core.Exceptions;
+using Auth0.ManagementApi.Models;
+using Auth0.Tests.Shared;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Auth0.ManagementApi.IntegrationTests
+{
+    public class LogStreamsTests : TestBase, IAsyncLifetime
+    {
+        private ManagementApiClient _apiClient;
+        private List<LogStream> createdStreams = new List<LogStream>();
+
+        public async Task InitializeAsync()
+        {
+            string token = await GenerateBruckeManagementApiToken();
+            _apiClient = new ManagementApiClient(token, GetVariable("BRUCKE_MANAGEMENT_API_URL"));
+        }
+
+        public Task DisposeAsync()
+        {
+            // Clean up any log stream entities on the tenant after every test executes
+            var deleteTasks = createdStreams.Select(stream => _apiClient.LogStreams.DeleteAsync(stream.Id));
+
+            return Task.WhenAll(deleteTasks.ToArray()).ContinueWith(_ =>
+            {
+                _apiClient.Dispose();
+            });
+        }
+
+        [Fact]
+        public async Task Test_log_stream_crud_sequence()
+        {
+            // Create a new entity
+            var name = "Auth0.net Test Log Stream";
+
+            var request = new LogStreamCreateRequest
+            {
+                Name = name,
+                Type = LogStreamType.Http,
+                Sink = new
+                {
+                    httpEndpoint = "https://my-stream.com",
+                    httpContentType = "application/json",
+                    httpContentFormat = "JSONOBJECT",
+                    httpAuthorization = "http-auth"
+                }
+            };
+
+            var createdLogStream = await _apiClient.LogStreams.CreateAsync(request);
+            createdStreams.Add(createdLogStream);
```

Comment:
Add the log stream to a collection so that it can be automatically deleted in `DisposeAsync`.